src/init.ts

```diff
@@ -84,8 +81,12 @@ async function generatePackageJson(
       }
     }
   }
+  return edits;
+}
 
-  // Install dev-dependencies.
+async function addDependencies(
```

Comment:
Same as above, think we shouldn't be explicit about the return type and just let type inference do it's thing